### ISSUE TITLE
Extracting more features

### ISSUE THREAD

**Issue by akim-ruslanov** [polyrhythm-project/polyrhythm-website#91]
Need to extract more features as listed:
1) Stress accent (found in ex 686)
2) Trill (found in ex 389)
3) Slur beginning (found in ex 129)
4) Slur ending (found in ex 724)

**Comment by epoudrier** [comment]
No. 1, stress accent refers to all possible tytpe of accent, including marcato, tenuto, staccato, etc.

**Comment by epoudrier** [comment]
Another category of accent that we discussed, which I refer as "second order" (as opposed to first-order, which create accents that are perceived more immediately as they are performed):

1. **Relatively long note** (i.e., note that is longer than the notes preceding and following it OR note that is longer than a computed average note length): Ex. 144 (dotted-eighth preceded and followed by sixteenth resulting in dotted-quarter against quarter); Ex. 198 (half against dotted-quarter, each being established through beginning duration accents); Ex. 584 (4-quarter melodic groups ending with a rest in winds against 3+2 quarter pattern in piano)
2. **Relatively large leap** (i.e., note following a melodic interval that is larger than a computed average interval)
3. **Contour accent** (i.e., note preceded by a change in direction): Ex. 396 (3-quarter in cello/bass against 5-quarter rhythmic pattern in violin/viola); Ex. 492 (3-quarter & 2-quarter patterns in piano RH & LH against 3+3+2 in strings)
